codegen/src/main/twirl/templates/PlayJavaServer/Router.scala.txt

```diff
@@ -27,26 +27,21 @@ import io.grpc.Status;
    * Abstract base class for implementing @serviceName in Java and using as a play Router
    */
   public abstract class Abstract@{serviceName}Router extends PlayRouter implements @{serviceName} {
-    private final Function<ActorSystem, Function<Throwable, Status>> eHandler;
-    private final ActorSystem system;
+    public scala.Function1<HttpRequest, Future<HttpResponse>> respond;
```

Comment:
I have 3 questions, should this be:
* public
* non-final
* scala.Function1 (in PlayJavaServer)
?

Reply:
You're right, changing public and non-final to `private final`.

`scala.Function1` seems OK since the user should never interact with it directly, and it's what's expected by the superclass `PlayRouter` (which is shared among Scala and Java API).